cmd/dp-dataset-exporter/main.go

```diff
@@ -56,11 +56,12 @@ func main() {
 	var serviceList initialise.ExternalServiceList
 
 	// Create kafka Consumer
-	kafkaConsumer, err := serviceList.GetConsumer(cfg)
+	kafkaConsumer, err := serviceList.GetConsumer(ctx, cfg)
 	exitIfError(ctx, err)
 
 	// Create kafka Producer
 	kafkaProducer, err := serviceList.GetProducer(
```

Comment:
Again dont shutdown app if cannot create a connection to kafka producer

Reply:
At this point, we only return an error if there is a channel validation error. If the Kafka broker(s) cannot be contacted we don't return an error, but the consumer/producer is not initialised. The health check will initialise it once the brokers are reachable.